gcc/rust/expand/rust-macro-builtins.cc

```diff
@@ -241,6 +241,7 @@ load_file_bytes (const char *filename)
   RAIIFile file_wrap (filename);
   if (file_wrap.get_raw () == nullptr)
     {
+      // TODO: is Location () ok?
```

Comment:
In that case, yes. We can't have locations, which point inside a file, if we can't open the file.
```suggestion
```